src/functionBased/About.js

```diff
@@ -18,7 +17,6 @@ const About = () => {
       </ul>
```

Comment:
These links don't work and they lead to `NoMatch` page with weird url addresses: 
![image](https://user-images.githubusercontent.com/87897753/149624116-36503eb6-f98f-40dc-9024-7e9d7f74e55e.png)
Could you fix them please?